app/constants.py

```diff
@@ -61,6 +65,10 @@ class PackCode(Enum):
     D_CE1A_ICLCR1 = 'D_CE1A_ICLCR1'
     D_CE1A_ICLCR2B = 'D_CE1A_ICLCR2B'
 
+    # Individual addressed initial contact letters for CE Units
```

Comment:
Only a comment but I think its the pack code for individual ICLs for CE estab and unit cases